blazor/datagrid/how-to/customize-column-styles.md

```diff
@@ -77,3 +77,56 @@ Add the custom CSS class to the specified column by using the [CustomAttributes]
 
 
 ![Customizing Column Style in Blazor DataGrid](../images/blazor-datagrid-column-style-customization.PNG)
+
+## Customize column header style
+
+You can customize the column header using the **e-headercell** CSS class.
+
+In the following sample, you can override the default style for column headers using the **e-headercell** CSS class.
```

Comment:
In the following sample, you can override the default style for column headers using **e-headercell** CSS class.